sql/telemetry_derived/addons_daily_v1/query.sql

```diff
@@ -0,0 +1,235 @@
+  STRUCT(
+    isoEngine(engine, 'google', v) AS google,
+    isoEngine(engine, 'bing', v) AS bing,
+    isoEngine(engine, 'ddg', v) AS ddg,
+    isoEngine(engine, 'yandex', v) AS yandex,
+    isoEngine(engine, 'amazon', v) AS amazon,
+    isoEngine(engine, 'other', v) AS other
+  )
+);
+
+WITH client_searches AS (
+  SELECT
+    client_id,
+    mapSearches(engine, sap) AS sap,
+    mapSearches(engine, tagged_sap) AS tagged_sap,
+    mapSearches(engine, tagged_follow_on) AS tagged_follow_on,
+    mapSearches(engine, organic) AS organic,
+    mapSearches(engine, searches_with_ads) AS searches_with_ads,
+    mapSearches(engine, ad_click) AS ad_clicks
+  FROM
+    (
+      SELECT
+        client_id,
+        CASE
+        WHEN
+          STARTS_WITH(engine, 'google')
+        THEN
+          'google'
+        WHEN
+          STARTS_WITH(engine, 'ddg')
+          OR STARTS_WITH(engine, 'duckduckgo')
+        THEN
+          'ddg'
+        WHEN
+          STARTS_WITH(engine, 'bing')
+        THEN
+          'bing'
+        WHEN
+          STARTS_WITH(engine, 'yandex')
+        THEN
+          'yandex'
+        WHEN
+          engine LIKE '%mazon%'
+        THEN
+          'amazon'
+        ELSE
+          'other'
+        END
+        AS engine,
+        SUM(COALESCE(sap, 0)) AS sap,
+        SUM(COALESCE(tagged_sap)) AS tagged_sap,
+        SUM(COALESCE(tagged_follow_on)) AS tagged_follow_on,
+        SUM(COALESCE(organic, 0)) AS organic,
+        SUM(COALESCE(ad_click, 0)) AS ad_click,
+        SUM(COALESCE(search_with_ads, 0)) AS searches_with_ads
+      FROM
+        `moz-fx-data-shared-prod.search.search_clients_daily`
+      WHERE
+        submission_date = @submission_date
+      GROUP BY
+        1,
+        2
+    )
+),
+client_meta AS (
+  SELECT
+    submission_date,
+    client_id,
+    os,
+    country,
+    subsession_hours_sum,
+    scalar_parent_browser_engagement_tab_open_event_count_sum,
+    scalar_parent_browser_engagement_total_uri_count_sum,
+    devtools_toolbox_opened_count_sum,
+    active_hours_sum,
+    days_since_seen,
+    sync_configured,
+    sap,
+    tagged_sap,
+    tagged_follow_on,
+    organic,
+    searches_with_ads,
+    ad_clicks,
+    active_addons
+  FROM
+    `moz-fx-data-shared-prod.telemetry.clients_last_seen`
+  LEFT JOIN
+    client_searches
+  USING
+    (client_id)
+  WHERE
+    submission_date = @submission_date
+),
+addons_expanded AS (
+  SELECT
+    *,
+    aa
+  FROM
+    client_meta
+  CROSS JOIN
+    UNNEST(active_addons) AS aa
+),
+daut AS (
+  SELECT
+    aa.addon_id,
+    COUNT(DISTINCT client_id) AS dau
+  FROM
+    addons_expanded
+  WHERE
+    days_since_seen = 0
+  GROUP BY
+    1
+),
+waut AS (
+  SELECT
+    aa.addon_id,
+    COUNT(DISTINCT client_id) AS wau
+  FROM
+    addons_expanded
+  WHERE
+    days_since_seen < 7
+  GROUP BY
+    1
+),
+maut AS (
+  SELECT
+    aa.addon_id,
+    COUNT(DISTINCT client_id) AS mau
+  FROM
+    addons_expanded
```

Comment:
```suggestion
    addons_expanded
  WHERE
    days_since_seen < 28
```

This is still correct without a WHERE clause, but I like having the 28 in here to be explicit about how we define mau.

Reply:
Agreed!